zebra-node-services/src/scan_service/response.rs

```diff
@@ -33,5 +30,5 @@ pub enum Response {
     ClearedResults,
 
     /// Response to `SubscribeResults` request
```

Comment:
###### Optional Suggestion

```suggestion
    /// Response to [`SubscribeResults`](super::request::Request::SubscribeResults) request
```

Reply:
Applied in https://github.com/ZcashFoundation/zebra/pull/8253/commits/b6f5461389d5af6a4c7e427f6a866b75a177b5c2.